RocketChat/Rocket.Chat
# [FIX] Omnichannel Activity Monitor closing chats returned to the queue

## Proposed changes (including videos or screenshots)


Fix `VisitorInactivityMonitor` is still monitoring rooms that returned to `Queue Chats`



## Issue(s)

 
## Steps to test or reproduce

1. Set the following 2 settings in `Admin -> Omnichannel -> Sessions` as below
![image](https://user-images.githubusercontent.com/34130764/115951668-8e8aaa00-a4ff-11eb-9903-142894577eec.png)
2. Start a new Livechat conversation
3. Send a message from Agent to visitor. Now within 20 seconds, move this chat back to queue.
4. At the end of 20 seconds, check the latest system message within the queued chat. You should see a system message saying that the chat was place on-hold

## Further comments
